library/proc_macro/src/quote.rs

```diff
@@ -135,6 +136,7 @@ pub fn quote(stream: TokenStream) -> TokenStream {
 /// Quote a `Span` into a `TokenStream`.
 /// This is needed to implement a custom quoter.
 #[unstable(feature = "proc_macro_quote", issue = "54722")]
-pub fn quote_span(_: Span) -> TokenStream {
-    quote!(crate::Span::def_site())
+pub fn quote_span(proc_macro_crate: TokenStream, span: Span) -> TokenStream {
```

Comment:
Even though this is unstable, how will this change affect the ecosystem? I'm guessing some nightly only crates will stop working until they update their deps, right?

Reply:
A search of Github didn't turn up any results (other than `library/proc_macro/src/quote.rs` in forks of rustc): https://github.com/search?l=Rust&p=1&q=quote_span&type=Code

I would be very surprised if anyone was using this:
1. It currently doesn't do anything, so it's easier to use manually use `Span::call_site()` or `Span::def_site()`
2. This is only useful if you're writing your own custom quote macro - as far as I know, the `quote` crate is the only custom macro there is, and it doesn't use `quote_span`.